src/relay/backend/compile_engine.cc

```diff
@@ -627,6 +627,7 @@ class CompileEngineImpl : public CompileEngineNode {
         const tvm::tir::StringImmNode* symbol_name = ext_symbol.as<tvm::tir::StringImmNode>();
         CHECK(symbol_name) << "No external symbol is set for:\n" << AsText(src_func, false);
         auto gv = GlobalVar(symbol_name->value);
+        src_func = WithAttr(src_func, attr::kCompiler, tir::StringImmNode::make("default"));
```

Comment:
Now you should not give it "default" but a null string ObjectRef, so that the returned value is not `defined`

https://github.com/apache/incubator-tvm/blob/0a0e58bfa4c87b2cbff0be2b401da0b3a08fcfe8/include/tvm/ir/function.h#L99

Reply:
Also, inorder to use CoW, you may want to use `src_func = WithAttr(std::move(src_func), ...)`